implementations/rust/ockam/ockam_identity/src/credential.rs

```diff
@@ -192,6 +192,9 @@ impl CredentialData<Unverified> {
     pub fn unverfied_key_label(&self) -> &str {
```

Comment:
I see why there's a typo in `unverfied_subject`. There's the same typo above 😄.

Reply:
Fixed all and references.